include/crocoddyl/core/integrator/euler.hxx

```diff
@@ -197,7 +182,10 @@ void IntegratedActionModelEulerTpl<Scalar>::quasiStatic(const boost::shared_ptr<
   // Static casting the data
   boost::shared_ptr<Data> d = boost::static_pointer_cast<Data>(data);
 
-  differential_->quasiStatic(d->differential, u, x, maxiter, tol);
+  VectorXs uc(control_->get_nu());
+  differential_->quasiStatic(d->differential, uc, x, maxiter, tol);
+  control_->params(controlData_, 0.0, uc);
+  u = controlData_->u_params;
```

Comment:
I cannot move the implementation in the parent class because the DifferentialActionData is introduced at the child class level. This is because in RK4 I need 4 instances of DifferentialActionData, so `d->differential` is a vector of DifferentialActionData in RK4, whereas it is a DifferentialActionData in Euler.